src/components/Songs.vue

```diff
@@ -7,7 +7,7 @@
     <b-table :items="songs" :fields="fields" hover v-on:row-clicked="clicked">
       <template slot="attributes.artwork" slot-scope="data">
         <lazy-img v-if="data.item.attributes && data.item.attributes.artwork"
-             :src="formatArtworkURL(data.item.attributes.artwork, 40, 40)" />
+             :src="data.value.artwork | formatArtworkURL(40)" />
```

Comment:
This should `data.item.attributes.artwork` instead of `data.value.artwork`.
Without it, the list of songs doesn't show since an error gets thrown.